getting-started/templates/systemlink-values.yaml

```diff
@@ -606,6 +606,8 @@ dataframeservice:
   kafka-ui:
     image:
       registry: *imageRegistryRef
+    imagePullSecrets:
+      - name: niartifacts-secret
```

Comment:
@lnowotny - I'm told you've dealt with other third-party charts that are expecting an array of maps instead of an array of strings like we use in our charts. Did you find another solution aside from forking the chart or copying the secret around?

If we do end up having to copy the secret for this release I'll ensure I update the release notes and file a bug to NIC2 to investigate a more long-term solution.

Reply:
If this requires special attention, it should have an ATTENTION comment.

Reply:
Does it require attention? The `global.imagePullSecrets` values doesn't specify an ATTENTION tag - based on that I assumed we didn't expect the customer to override this secret name.

Reply:
But if they do update that value, then their deployment will be broken. In such cases, I've tried to add ATTENTION tags at the relevant point where something will break.